src/tests/testService.ts

```diff
@@ -392,30 +392,40 @@ export class TestService {
     return { tests: classItems, testLevel: TestLevel.RunSpecifiedTests };
   }
 
-  private async buildTestPayload(
+  public async buildTestPayload(
```

Comment:
Instead of changing this to public, can you try casting the service to any in the test to access this method?

ie

`(testService as any).buildTestPayload(tests);`

Reply:
Sure. Nice! Thanks